code/modules/admin/admin_verbs.dm

```diff
@@ -873,6 +874,8 @@ GLOBAL_PROTECT(admin_verbs_poll)
 		// Finally, ensure the minds are tracked and in the manifest.
 		SSticker.minds += character.mind
 		if(ishuman(character))
+			GLOB.player_list += character
+			GLOB.joined_player_list += character
```

Comment:
...... I very much do not like this. It breaks the promise of that list, and will very likely just hard delete to shit

Reply:
Fair enough. I'm not super confident in my understanding the global lists, but getting them on these lists made testing stuff for events easier. Is there a better way of getting the dummy crew on the list, or is the problem that they're being added to it at all?

Reply:
its a list of players with clients. debug crew does not have clients to operate on. since the list is of players with clients. this means we assume that everyone there has a client, but this will not be the case with your change, and so it will runtime when it tries to fetch something from a nonexistent client